test/grant_general_action_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+describe AccessControl do
+
+  it 'returns nil after a successful grant' do
+    actor = User.create!
+
+    assert_nil(AccessControl::Query.new(actor).grant(1, Post))
```

Comment:
I missed this before, but we're using `must_equal` in other tests and `assert_stuff` here. It feels like we should standardize one way or the other.

Preferences? I think it's fine either way. I'm just looking for consistency.

Reply:
I had `must_equal nil` -- but there were a bunch of deprecation warnings about that and the suggested remedy was to use `assert`